components/i18n/languages/fr_FR.ts

```diff
@@ -14,6 +14,8 @@ export default {
     filterConfirm: 'OK',
     filterReset: 'Réinitialiser',
     emptyText: 'Aucune donnée',
+      selectAll: 'tout sélectionner',
```

Comment:
Indent

Reply:
OK and thanks for your advice.

Reply:
Oh. And the first character should be uppercase. Didn't noticed that. Sorry. 🤣

Reply:
My mistake. See if it's ok now.